source/common/grpc/common.cc

```diff
@@ -24,6 +24,22 @@ namespace Grpc {
 
 const std::string Common::GRPC_CONTENT_TYPE{"application/grpc"};
 
+Status::GrpcStatus Common::getGrpcStatus(const Http::HeaderMap& trailers) {
+  const Http::HeaderEntry* grpc_status_header = trailers.GrpcStatus();
+
+  uint64_t grpc_status_code;
+  if (!grpc_status_header || grpc_status_header->value().empty()) {
+    return Status::GrpcStatus::MissingStatus;
```

Comment:
I'm inclined to make this an optional return type, since having a `GrpcStatus` that indicates status missing is somewhat self contradictory.